virttest/test_setup.py

```diff
@@ -439,6 +439,12 @@ def __init__(self, params):
             target_hugepages = int(target_hugepages)
 
         self.target_hugepages = target_hugepages
+        self.target_nodes = params.get("target_nodes")
```

Comment:
let's have target_nodes, target_num_node0 params in base.cfg and explain its usage

Reply:
@balamuruhans Good idea!  Please help check if the explanation is clear. Thanks!